### PR TITLE
Document load balancer

### LINKED ISSUE #235: document load balancer
and link it into the quickstart download page

Link to it from the cluster configuration pages

### PR DESCRIPTION
**Pull request by floren**
Closes #235

### CODE REVIEW

File: distributed/loadbalancer.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# The Gravwell Load Balancer
+
+To make setting up your environment as easy as possible, Gravwell provides a custom load balancer specifically designed for use with Gravwell webservers. It can automatically discover Gravwell webservers, meaning you don't need to reconfigure the loadbalancer every time you add or remove a webserver--and if a webserver goes down, it will automatically direct users to another server.
+
+## Load Balancer Architecture
+
+The load balancer is an HTTP(S) proxy which automatically directs clients to one of the Gravwell webservers. It sets a cookie on the user's browser to maintain a level of "stickiness", so one session's requests all go to the same webserver.
+
+The load balancer discovers Gravwell webservers by communicating with the Gravwell  (see [the distributed webserver documentation](frontend.md) for more info about the datastore).
```
Comment (traetox) [inline distributed/loadbalancer.md:9]: with the Gravwell.... what?

**Review by traetox — CHANGES_REQUESTED** [review]
weird wording when talking about the datastore.  Am i missing something?

File: distributed/loadbalancer.md (hunk 1 above)
Comment (floren) [reply distributed/loadbalancer.md:9]: Forgot to finish the sentence. Stand by. I'll also fix the typo you introduced a minute ago ;)

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**